src/stats.php

```diff
@@ -108,6 +108,16 @@ function getGitHubApiResponse(string $url): string
     curl_setopt($ch, CURLOPT_VERBOSE, false);
     curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, true);
     $response = curl_exec($ch);
+
+    //Handles curl errors
+    if($response==false){
+        if(str_contains(curl_error($ch),'unable to get local issuer certificate'))
+        {
+            throw new InvalidArgumentException("You don't have valid SSL Certificate installed or XAMPP.");
+        }
+        throw new InvalidArgumentException("Something is wrong with getGitHubApiResponse().");
+    }
```

Comment:
Formatting and triple equals for comparing with false literal

```suggestion
    // Handles curl errors
    if ($response === false) {
        if (str_contains(curl_error($ch), "Unable to get local issuer certificate.")) {
            throw new InvalidArgumentException("You don't have a valid SSL Certificate or XAMPP installed.");
        }
        throw new InvalidArgumentException("An error occurred in getGitHubApiResponse().");
    }
```